recipe/meta.yaml

```diff
@@ -16,25 +16,14 @@ source:
     - fix_ppc64le_set_mempolicy_home_node.patch  # [ppc64le]
 
 build:
-  number: 2
+  number: 3
   skip: true  # [not linux]
 
-requirements:
-  build:
-    - automake
-    - autoconf
-    - coreutils
-    - grep
-    - libtool
-    - sed
-    - {{ compiler('c') }}
-    - {{ stdlib('c') }}
-    - make
```

Comment:
I don't think we can remove these since we use a top level `build.sh` which will expect to run in the `numa-split` build environment